Tests/ArcGISToolkitTests/PreplannedMapModelTests.swift

```diff
@@ -282,40 +268,87 @@ class PreplannedMapModelTests: XCTestCase {
             preplannedMapAreaID: areaID
         )
         
-        model2.status.assertExpectedValue(.downloaded)
+        XCTAssertEqual(model2.status, .downloaded)
+    }
+    
+    @MainActor
+    func testLoadMobileMapPackage() async throws {
+        let portalItem = PortalItem(portal: Portal.arcGISOnline(connection: .anonymous), id: .init("acc027394bc84c2fb04d1ed317aac674")!)
+        let task = OfflineMapTask(portalItem: portalItem)
+        let areas = try await task.preplannedMapAreas
+        let area = try XCTUnwrap(areas.first)
+        let areaID = try XCTUnwrap(area.portalItem.id)
+        
+        defer {
+            // Clean up JobManager.
+            OfflineManager.shared.jobManager.jobs.removeAll()
+            
+            // Clean up folder.
+            let directory = FileManager.default.preplannedDirectory(
+                forPortalItemID: portalItem.id!,
+                preplannedMapAreaID: areaID
+            )
+            try? FileManager.default.removeItem(at: directory)
+        }
+        
+        let model = PreplannedMapModel(
+            offlineMapTask: task,
+            mapArea: area,
+            portalItemID: portalItem.id!,
+            preplannedMapAreaID: areaID,
+            // User notifications in unit tests are not supported, must pass false here
+            // or the test process will crash.
+            showsUserNotificationOnCompletion: false
+        )
+        
+        var statuses: [PreplannedMapModel.Status] = []
```

Comment:
This resembles the `AsyncAccumulator` pattern in our SDK test suite. Maybe later we can change it to that. No need to change here.